src/evaluate/evaluator/utils.py

```diff
@@ -16,3 +18,14 @@ def __getitem__(self, i):
 
     def __iter__(self):
         return (self.dataset[i][self.key] for i in range(len(self)))
+
+
+def choose_split(data):
+    dataset_splits = requests.get(f"https://datasets-server.huggingface.co/splits?dataset={data}").text
+    available_splits = [split["split"] for split in ast.literal_eval(dataset_splits)['splits']]
```

Comment:
a simpler solution is potentially using `from datasets import get_dataset_split_names`:
```suggestion
    available_splits = get_dataset_split_names(data)
```